Libraries/LibWeb/CSS/Parser/ValueParsing.cpp

```diff
@@ -3769,10 +3769,16 @@ bool Parser::substitute_attr_function(DOM::Element& element, FlyString const& pr
     // 1. If the attr() function has a substitution value, replace the attr() function by the substitution value.
     // https://drafts.csswg.org/css-values-5/#attr-types
     if (element.has_attribute(attribute_name)) {
+        auto parse_string_as_component_value = [&](String const& string) {
```

Comment:
boo, [&] capture. Can you explicitly capture just stream by reference instead? `[&stream]`